airrohr-firmware/airrohr-firmware.ino

```diff
@@ -4225,11 +4225,15 @@ static bool acquireNetworkTime() {
 #if defined(ESP8266)
 	settimeofday_cb(time_is_set);
 #endif
-	configTime(8 * 3600, 0, "pool.ntp.org");
+	configTime(0, 0, "pool.ntp.org"); // using UTC as timezone
 	while (retryCount++ < 20) {
-		// later than 2000/01/01:00:00:00
+		now = time(nullptr);		
+#if defined(ESP8266)
 		if (sntp_time_is_set) {
-			now = time(nullptr);
+#endif
+#if defined(ESP32)
+		if (now > 17897*24*60*60) { // later than 2019/01/01:00:00:00
```

Comment:
why is sntp_time_is_set now only used for ESP8266?

Reply:
because settimeofday_cb() is not available for ESP32. The examples are using sntp_set_time_sync_notification_cb() but this is not yet available in the newest esp32platform delivered with platformio.

Reply:
Agreed, thanks